internal/config/audit.go

```diff
@@ -58,6 +58,7 @@ func (c *AuditConfig) validate() error {
 // SinksConfig contains configuration held in structures for the different sinks
 // that we will send audits to.
 type SinksConfig struct {
+	Events  []string          `json:"events,omitempty" mapstructure:"events"`
```

Comment:
We could add a default to the `*AuditConfig.setDefaults` to set the events default to `["*:*"]`.
That way this is not a breaking change.